native/src/components/AddressInfo.tsx

```diff
@@ -48,7 +48,7 @@ const AddressInfo = ({ location, language }: AddressInfoProps): ReactElement =>
 
   return (
     <Container language={language}>
-      <Pressable role='button' onPress={copyLocationToClipboard}>
+      <Pressable accessibilityLabel={t('cityNotCooperating:copyText')} role='button' onPress={copyLocationToClipboard}>
```

Comment:
:wrench: I think `Copy text` is not a good a11y label here, I think it should explicitly say `Copy address`. What do you think?

Reply:
I think this depends on whether we want to add more translations.

Reply:
Adding new translations is totally fine if it makes sense. I think in this case it is better to use a new and correct translation over a more or less incorrect (or at least imprecise) one.